docs/source/defining-parameters.rst

```diff
@@ -222,8 +222,8 @@ We can do the following:
 
 .. code:: python
 
-    reference_data_path = '/global/cfs/cdirs/e3sm/acme_diags/obs_for_acme_diags/'
-    test_data_path = '/global/cfs/cdirs/e3sm/acme_diags/test_model_data_for_acme_diags/'
+    reference_data_path = '/global/cfs/cdirs/e3sm/e3sm_diags/obs_for_acme_diags/'
+    test_data_path = '/global/cfs/cdirs/e3sm/e3sm_diags/test_model_data_for_acme_diags/'
```

Comment:
same as above

Reply:
wontfix based on decision to use symlinks from `acme_diags` to `e3sm_diags`.